test/mitmproxy/net/test_tcp.py

```diff
@@ -611,7 +600,6 @@ def test_dhparams(self):
 
 
 class TestTCPClient(tservers.ServerTestBase):
-
```

Comment:
Please don't delete all those lines.

Reply:
Actually, I am trying to create a new class so I will not make any changes to existing one.